office/charm/Portfile

```diff
@@ -29,19 +29,20 @@ subport ${name}-qt5 {
 if {${subport} ne "${name}-qt5"} {
     PortGroup       qt4 1.0
 
-    revision        20151018
+    revision        20200207
 
     conflicts       ${name}-qt5
     depends_build-append \
                     port:automoc
     depends_lib-append \
-                    port:qt4-mac-sqlite3-plugin
+                    port:qt4-mac-sqlite3-plugin port:qtkeychain-qt5
```

Comment:
This won't work, I suppose you meant to add the qtkeychain-qt5 dependency to the charm-qt5 subport?!

Reply:
you are right!
So I moved this dep to -qt5 version and changed this one to qt4.